exporters/trace/jaeger/jaeger.go

```diff
@@ -68,9 +76,24 @@ func WithBufferMaxCount(bufferMaxCount int) func(o *options) {
 	}
 }
 
-// NewExporter returns a trace.Exporter implementation that exports
+// WithSDK sets the SDK config for the exporter pipeline.
+func WithSDK(config *sdktrace.Config) func(o *options) {
+	return func(o *options) {
+		o.Config = config
+	}
+}
+
+// WithRegistration enables the registration of the trace provider of the new pipeline
+// as Global Trace Provider.
+func WithRegistration() func(o *options) {
```

Comment:
Similar to changing the option name, we should change this to `WithRegisterGlobal`, right?

Or, maybe `RegisterAsGlobal`?

Reply:
RegisterAsGlobal() seems better.